curriculum/challenges/chinese/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-with-javascript-for-loops.md

```diff
@@ -15,7 +15,7 @@ JavaScript 中最常见的循环就是 `for`，它可以循环指定次数。
 
 for 循环中的可选三个表达式用分号隔开：
 
-`for (a; b; c)`，其中 `a` 为初始化语句，`b` 是循环条件语句，`c` 是终止循环条件表达式。
+`for (a; b; c)`，其中`a`为初始化语句，`b`为条件语句，`c` code> 是最终的表达式。
```

Comment:
This looks malformed - Crowdin bug

Reply:
![image](https://user-images.githubusercontent.com/63889819/140614743-fa6fc7e2-e506-4392-82ab-39ad8b89824e.png)

Not crowdin bug.